### PR TITLE
Vision 1.1 API Client

### PR DESCRIPTION
**Pull request by lukesneeringer**
Final merge of the Vision 1.1 API client.
Needs CI to complete (and possibly the 0.15.2 GAPIC to propagate).

### CODE REVIEW

File: docs/vision-usage.rst [hunk 1]
```diff
@@ -317,6 +341,43 @@ Multiple images can be processed with a single request by passing
     <Likelihood.VERY_LIKELY: 'POSSIBLE'>
 
 
+***************
+Web Annotations
```
Comment (daspecster) [inline docs/vision-usage.rst]: This should probably be 'Web Detection` right?

File: tox.ini [hunk 2]
```diff
@@ -270,6 +270,8 @@ commands =
     {[testing]localdeps}
     python {toxinidir}/system_tests/attempt_system_tests.py {posargs}
 deps =
+    {toxinidir}/_vendor/proto-google-cloud-vision-v1
```
Comment (daspecster) [inline tox.ini]: Also these need to go.

Comment (daspecster) [inline tox.ini]: This is why Travis failed actually.

**Comment by daspecster** [comment]
Fixing the lint issue.

**Comment by daspecster** [comment]
I'm not sure what's going on with Travis-pr. I cleared the caches for it I think. Once the Travis-push turns green I'll delete the other caches and try Travis-pr again.

**Comment by daspecster** [comment]
Deleted caches and restarted travis.

**Comment by daspecster** [comment]
@lukesneeringer I think you force pushed and removed my commits before.

**Comment by lukesneeringer** [comment]
Yeah, I did. Sorry, I thought I was plowing over one of mine.

**Comment by lukesneeringer** [comment]
I am waiting on the last Travis run to finally pass (even though it is running twice sequentially due to the way I pushed).

**Comment by lukesneeringer** [comment]
Finally done. (And docs even build!)

**Comment by jonparrott** [comment]
@gguuss are you taking on samples for this?

**Comment by gguuss** [comment]
@jonparrott correct, the samples are ready.

**Comment by lukesneeringer** [comment]
Am I correct to assume this means docs are updated as well?

**Pull request merged**